generate/testdata/queries/Pointers.graphql

```diff
@@ -1,11 +1,14 @@
 # @genqlient(pointer: true)
+# @genqlient(for: "UserQueryInput.id", pointer: false)
+# @genqlient(for: "User.id", pointer: false)
 query PointersQuery(
```

Comment:
I think it would be good to put those example in the docs. All your examples have just one directive and I think it's not necessarily obvious how they combine.   This examples makes it very clear.